dialog-el.html

```diff
@@ -16,13 +16,14 @@
       border-radius: 4px;
       box-shadow: 0px 0px 4px 0px rgba(0,0,0,0.35), 0px 3px 2px 0px rgba(0,0,0,0.18);
       opacity: 0;
-      padding: 15px;
+      padding: 10px;
       position: relative;
       -webkit-transform: scale(0.7);
       transform: scale(0.7);
       -webkit-transition: transform 0.3s, opacity 0.3s;
       transition: transform 0.3s, opacity 0.3s;
       will-change: transform, opacity;
+      @apply(--dialog-el-content);
```

Comment:
I think that should be alright. Then I can remove the padding change I made too.